recipes/range-v3/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.5)
+project(test_package)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
can we change this to use the find_pacakge() approach and `cmake_find_package_multi` to try something different? I think it makes sense for this header only library

Reply:
sure!

Reply:
done!